cli/src/migrations/schema.ts

```diff
@@ -48,16 +48,22 @@ export const migrationFile = z.object({
   operations: z.array(migrationOperation)
 }) satisfies z.ZodType<Schemas.MigrationObject>;
 
+export const pgRollMigrationHistoryObject = z.object({
+  name: z.string(),
+  operations: z.array(z.any())
+});
```

Comment:
I'm fine with the `any` for now, do you think it makes sense to add `@xata.io/pgroll` as a dependency and import the generated zod schemas?

Reply:
I added it :)